### ISSUE TITLE
measurements/history using wrong default endTime

### ISSUE THREAD

**Issue by juliusmarminge** [sensor-network/open-data-portal#90]
__Describe the bug:__
The default endTime parameter is not set to current time

__To reproduce:__
Navigate to `http://<SERVER_URL>/api/v3/measurements/history`. 

__Observed behavior:__
Per the API documentation, the `endTime` should default to `current time` which is not the case:

```json
{
  "summary": {
    "locationName": "Everywhere",
    "startTime": "2022-01-01",
    "endTime": "2022-04-29",
    "sensors": {
      ...
    }
  },
  "measurements": [
    ...
  ]
}
```

__Expected behavior:__
```json
{
  "summary": {
    "locationName": "Everywhere",
    "startTime": "2022-01-01",
    "endTime": "2022-05-05",  // at the time of reporting
    "sensors": {
      ...
    }
  },
  "measurements": [
    ...
  ]
}
```

__Environment:__
Server: latest production build
Browser: Safari, Chrome

**Comment by juliusmarminge** [comment]
@andreas-bauer I can't seem to reproduce this issue locally.
Result when querying `http://localhost:3000/api/v3/measurements/history`
```json
{
  "summary": {
    "locationName": "Everywhere",
    "startTime": "2022-01-01",
    "endTime": "2022-05-05",
    ...
}
```
Do you think this could be an issue with your server-settings or the docker-setup you have?

**Comment by juliusmarminge** [comment]
Fixed by #92.

Now keeping all the time information as Date objects instead of switching back and forth from ISO strings and Date objects.
